libcontainer/cgroups/systemd/dbus.go

```diff
@@ -0,0 +1,66 @@
+package systemd
+
+import (
+	"sync"
+
+	systemdDbus "github.com/coreos/go-systemd/v22/dbus"
+)
+
+type dbusConnManager struct {
+	conn     *systemdDbus.Conn
+	rootless bool
+	mut      sync.RWMutex
```

Comment:
You can include it anonymously, i.e.
```suggestion
	sync.RWMutex
```

and use `d.RLock()` etc below.